eng/pipelines/templates/jobs/ci.yml

```diff
@@ -189,6 +189,7 @@ jobs:
           arguments: >
             -SourceDirectory $(System.DefaultWorkingDirectory)/build
             -TargetDirectory $(Build.ArtifactStagingDirectory)
+            -Artifacts ('${{ replace(convertToJson(parameters.Artifacts), '''', '`''') }}' | ConvertFrom-Json | Where-Object -Not skipPublishPackage )
```

Comment:
@JimSuplizio @hallipr should this also be filtering on releaseInBatch similar to https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/archetype-sdk-client.yml#L133 so we control what packages get uploaded, signed, and published?

Reply:
I'm not sure.   I don't know the purpose of staging here